.github/workflows/pr-tests.yaml

```diff
@@ -3,7 +3,7 @@
 
 # We should restore the cache and remove extra requirements installations after this is resolved https://github.com/actions/setup-python/issues/182
 
-name: PR Tests
+name: Build
```

Comment:
Should the file name be updated as well? Still build seems a bit counterintuitive to me, a build usually generates an artifact. A docker build, static files for a website or a compiled binary. But in this case it's testing